assets/src/blocks/Submenu/SubmenuBlock.js

```diff
@@ -30,6 +29,10 @@ export class SubmenuBlock {
           type: 'string',
           default: ''
         },
+        submenu_style: { // Needed for old blocks conversion
```

Comment:
We could also keep the deprecated entry if we would only add the field here to not break older blocks?